test/e2e/cypress/integration/profiles/code/registration/success.spec.ts

```diff
@@ -109,29 +114,28 @@ context("Registration success with code method", () => {
         cy.setPostCodeRegistrationHooks([])
         const email = gen.email()
 
-        cy.get(` input[name='traits.email']`).type(email)
+        cy.get(`input[name='traits.email']`).type(email)
 
         cy.submitCodeForm()
+        cy.get('[data-testid="ui/message/1040005"]').should(
+          "contain",
+          "An email containing a code has been sent to the email address you provided",
+        )
 
-        cy.url().should("contain", "registration")
         cy.getRegistrationCodeFromEmail(email).should((code) => {
-          cy.get(` input[name=code]`).type(code)
+          cy.get(`input[name=code]`).type(code)
           cy.get("button[name=method][value=code]").click()
         })
 
-        cy.deleteMail({ atLeast: 1 })
```

Comment:
All methods like `cy.getRegistrationCodeFromEmail` remove mail automatically. This condition leads to us waiting for about a minute to delete an email, which then times out (and passes silently) because no email was found.